packages/sdk/src/types.ts

```diff
@@ -68,6 +68,11 @@ export enum GRAPHQL_ENDPOINTS  {
   testnet = 'https://interop-testnet.hasura.app/v1/graphql',
 }
 
+export enum FASTNEAR_RPC_ENDPOINTS  {
+  mainnet = 'https://free.rpc.fastnear.com/',
+  testnet = 'https://g.w.lavanet.xyz:443/gateway/neart/rpc-http/f538cb3b0a85aafdb9996886d004ee0a',
+}
+
```

Comment:
Isn't this a public project? We have hardcoded api keys here.

Reply:
Already existed before: https://github.com/Mintbase/mintbase-js/pull/521/files#diff-9d8c1fba9e81b9e94776fadecf0793c4bbacdf4474689eb6817576f5f7e6b5a8R76-R78

IIRC we did that knowing that we could deprecate the key and get a new one in case it gets abused.